.github/workflows/sample_data.yaml

```diff
@@ -35,5 +35,5 @@ jobs:
           sudo systemctl stop puma sidekiq
           mkdir -p ~/backup/
           pg_dump -h localhost -U ofn_user openfoodnetwork | gzip > ~/backup/staging-`date +%Y%m%d%H%M%S`-before-reset.sql.gz
-          bundle exec rake db:reset ofn:sample_data
+          bundle exec rake db:reset ofn:${{ secrets.sample_data_name }}
```

Comment:
I am not sure I am following this, as far as I understand we only have `ofn:sample_data` task for sample data ?
What are the expected inputs here ?